python/tvm/relay/op/transform.py

```diff
@@ -611,31 +611,41 @@ def split(data, indices_or_sections, axis=0):
     return TupleWrapper(_make.split(data, indices_or_sections, axis), ret_size)
 
 
-def strided_slice(data, begin, end, strides=None):
+def strided_slice(data, begin, end, strides=None, slice_mode=False):
     """Strided slice of an array.
 
     Parameters
     ----------
     data : relay.Expr
         The source array to be sliced.
 
-    begin: list of int
+    begin: relay.Expr or List[int]
         The indices to begin with in the slicing.
 
-    end: list of int
+    end: relay.Expr or List[int]
         Indices indicating end of the slice.
 
-    strides: list of int, optional
+    strides: relay.Expr or List[int], optional
         Specifies the stride values, it can be negative in that case,
         the input tensor will be reversed in that particular axis.
 
+    slice_mode: boolean, optional
+        Whether to ignore the negative elements in input end,
+        will slice to the end of data for the ignored element.
```

Comment:
Provide more details about this attribute, such as end is actually slice size and strides is ignored.